src/Routes/ScrollToTop.ts

```diff
@@ -0,0 +1,11 @@
+import { useEffect } from 'react'
```

Comment:
is `src/Routes` the right place for this? This looks like like a `lib` thing

Reply:
i think this is ok because it's more reacty than datay/libby